src/dialects/postgres/string.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
```

Comment:
This should be made to be inherited from `src/query/string.js`, and both, base and inherited instance should be refactored to only override small part of class in inherited implementation. (SqlString must be instantiated as well to make inheritance work nice)